spacy/pipeline/transition_parser.pyx

```diff
@@ -665,4 +665,4 @@ cdef class Parser(TrainablePipe):
                     golds.append(gold)
                 if state.is_final():
                     break
-        return states, golds, max_length
+        return states, golds, max_length
```

Comment:
I think this change can just be reverted

Reply:
Fixed.